example/docker-compose/scalable-single-binary/tempo-scalable-single-binary.yaml

```diff
@@ -30,6 +30,9 @@ ingester:
     heartbeat_period: 100ms
 
 compactor:
+  ring:
```

Comment:
ah, is the "fix" just having a correct example?

i thought we defaulted to memberlist if no config was present? maybe we should do that?

Reply:
#1613 has been filed from our recent meeting.  The fix was indeed just to include the ring config in the examples.